src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsEvolveOasis.java

```diff
@@ -8,42 +8,42 @@
 
 import org.testng.annotations.Test;
 
-public class TestEvolveAds extends TemplateNoFirstLoad {
+public class TestAdsEvolveOasis extends TemplateNoFirstLoad {
 
   @GeoEdgeBrowserMobProxy(country = "CA")
   @UseUnstablePageLoadStrategy
   @Test(
       dataProviderClass = AdsDataProvider.class,
-      groups = {"TestEvolveAds"},
+      groups = "AdsEvolveOasis",
       dataProvider = "evolveTestPage"
   )
-  public void testEvolveCall_CA(String wikiName, String article) {
-    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
-    AdsEvolveObject wikiPage = new AdsEvolveObject(driver, testedPage);
-    wikiPage.verifyEvolveCall();
+  public void adsEvolveOasisCA(String wikiName, String article) {
+    adsEvolveOasis(wikiName, article);
   }
 
   @GeoEdgeBrowserMobProxy(country = "AU")
```

Comment:
I analysed 2 approaches and @justnpT's approach looks "less overkill" to [me](https://github.com/Wikia/selenium-tests/blob/5313b07deb3a6a38bb438d04a2c3cfe0ea755dfd/src/test/java/com/wikia/webdriver/common/core/geoedge/CountryCode.java) :smirk_cat: 

2 points why:
* we use *only* country codes (ISO 3166-1 alpha-2 code) and country names
* country list grew only on 3-5 countries during last 1.5 years - not a big deal

Thanks @justnpT, the change truly increase readability :wink: 

---

Another *bigger* issue related to GeoEdge.
We have 2 ways to set-up GeoEdge:
* via [annotation](https://github.com/Wikia/selenium-tests/blob/5313b07deb3a6a38bb438d04a2c3cfe0ea755dfd/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsScreenshotComparisonMobile.java#L35)
* via [config](https://github.com/Wikia/selenium-tests/blob/5313b07deb3a6a38bb438d04a2c3cfe0ea755dfd/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsMonitoringOasis.java#L22)

(we need to think which approach is better and stick to one of them)